CHANGES/650.contrib

```diff
@@ -1,3 +1,2 @@
-The pull request template to be consistent with :ref:`the contributing guidelines
-<adding change notes with your prs>`, particularly the list of categories
+The pull request template to be consistent with :ref:`the contributing guidelines <adding change notes with your prs>`, particularly the list of categories
```

Comment:
without this change the linter was failing.

Reply:
How?

Reply:
Same as CI: https://github.com/aio-libs/frozenlist/actions/runs/19160278857/job/54769109511

```
Checking dist/frozenlist-1.8.1.dev0-py3-none-any.whl: FAILED
ERROR    `long_description` has syntax errors in markup and would not be        
         rendered on PyPI.                                                      
         line 146: Warning: Inline literal start-string without end-string.     
Checking dist/frozenlist-1.8.1.dev0.tar.gz: FAILED
ERROR    `long_description` has syntax errors in markup and would not be        
         rendered on PyPI.                                                      
         line 146: Warning: Inline literal start-string without end-string.     
Error: Process completed with exit code 1.
```